 hwui: Remove deprecated compiler flag

 * HWUI_COMPILE_FOR_PERF uses the -mapcs flag
   but it has already been deprecated.
 * Remove the flag "-mapcs" completely

diff --git a/libs/hwui/Android.bp b/libs/hwui/Android.bp
index 0089da4..c6f8b9a 100644
--- a/libs/hwui/Android.bp
+++ b/libs/hwui/Android.bp
@@ -4,7 +4,7 @@ cc_defaults {
         "hwui_static_deps",
 
         //"hwui_bugreport_font_cache_usage",
-        //"hwui_compile_for_perf",
+        "hwui_compile_for_perf",
     ],
 
     cflags: [
@@ -84,7 +84,6 @@ cc_defaults {
     cflags: [
         "-fno-omit-frame-pointer",
         "-marm",
-        "-mapcs",
     ],
 }
 
